ynr/apps/elections/api/next/serializers.py

```diff
@@ -82,6 +83,27 @@ def get_ballots(self, obj):
         ).data
 
 
+class ResultSetSerializer(serializers.ModelSerializer):
```

Comment:
This seemed fairly straightforward after discussing with @GeoWill this morning so I don't know where I've gone wrong. I've also tried to put `ResultSetSerializer` in ` ynr/apps/uk_results/api/next/serializers.py` but then realised that wasn't the endpoint I was after so moved it here. 

I've made sure to check the ballot for a locked ballot with results entered.